pom.xml

```diff
@@ -8,11 +8,11 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <scala.version>2.11.8</scala.version>
-    <scala.main.version>2.11</scala.main.version>
-    <spark.version>2.4.8</spark.version>
+    <scala.version>2.12.17</scala.version>
+    <scala.main.version>2.12</scala.main.version>
+    <spark.version>3.1.3</spark.version>
     <scalatest.version>3.2.12</scalatest.version>
-    <connector.version>2.5.2</connector.version>
+    <connector.version>3.1.0</connector.version>
     <cassandra.version>3.11.13</cassandra.version>
```

Comment:
[`3.11.14` is the latest 3.x series and `4.0.7` is the latest in 4.x series](https://cassandra.apache.org/_/download.html) . Should we change it?

Reply:
sounds good to me